lib/server.ts

```diff
@@ -64,12 +64,14 @@ function allAddresses (type): string[] {
         family = 'IPv6'
     }
     const addresses: string[] = []
+    const macs: string[] = []
     const interfaces = os.networkInterfaces()
     for (const ifname in interfaces) {
         if (ifname in interfaces) {
             interfaces[ifname]?.forEach((a) => {
-                if (a.family === family) {
+                if (a.family === family && macs.includes(a.mac) == false) {
```

Comment:
Thank you, the changes look good to me! :)

As indicated by the linter, though, this line should probably be changed as follows:

```suggestion
                if (a.family === family && !macs.includes(a.mac)) {
```